rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -273,9 +257,83 @@ boolean killJob(@HeaderParam("sessionid")
     @GET
     @Path("jobs/{jobid}/tasks")
     @Produces("application/json")
-    List<String> getJobTasksIds(@HeaderParam("sessionid")
-    String sessionId, @PathParam("jobid")
-    String jobId) throws NotConnectedRestException, UnknownJobRestException, PermissionRestException;
+    List<String> getJobTasksIds(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId)
+            throws NotConnectedRestException, UnknownJobRestException, PermissionRestException;
+
+    /**
+     * Returns a list of the name of the tasks belonging to job <code>jobId</code> with pagination
+     * @param sessionId a valid session id
+     * @param jobId jobid one wants to list the tasks' name
+     * @param offset the number of the first task to fetch
+     * @param limit the number of the last task to fetch (non inclusive)
+     * @return a list of tasks' name
+     */
+    @GET
+    @Path("jobs/{jobid}/tasks/paginated")
+    @Produces("application/json")
+    List<String> getJobTasksIdsPaginated(@HeaderParam("sessionid") String sessionId,
```

Comment:
Is the method returning task names? if yes, why not a more simple name like ` getTaskNamePaginated` or `getTaskInformationPaginated`?